apps/passport-client/new-components/Typography.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styled from "styled-components";
+
+const TypographyText = styled.span`
```

Comment:
slightly unclear what this is for - typography is very vague as a name. are you just trying out the font here? if so that's fine but leave a comment

Reply:
not only, but also define a set of sizes and styles for our texts,